### PR TITLE
Creating small docker container for GATK 3.5

### PR DESCRIPTION
**Pull request by apaul7**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ RUN apt-get update -y && apt-get install -y \
     wget
 
 RUN cd /tmp/ \
-    && wget -O /tmp/gatk3.6.tar.bz2 'https://software.broadinstitute.org/gatk/download/auth?package=GATK-archive&version=3.6-0-g89b7209' \
-    && tar xf gatk3.6.tar.bz2 \
-    && cp GenomeAnalysisTK.jar /opt/GenomeAnalysisTK.jar \
+    && wget -O /tmp/gatk3.5.tar.bz2 'https://software.broadinstitute.org/gatk/download/auth?package=GATK-archive&version=3.5-0-g36282e4' \
+    && tar xf gatk3.5.tar.bz2 \
+    && cp GenomeAnalysisTK.jar /opt/GenomeAnalysisTK-3.5.jar \
```
Comment (tmooney) [inline Dockerfile]: If we're going to have a 3.5-specific container, we could keep this at `/opt/GenomeAnalysisTK.jar`.

**Review by tmooney — COMMENTED** [review]
This should probably push to a `3.5.0` branch instead of `master`--it seems a little backwards to have an older version become `latest`.

**Comment by apaul7** [comment]
ahh darn. Then to make a new branch I need to do that when I create the PR? Or can I just create a new branch in this repo and alter the PR to go to that branch?

**Comment by tmooney** [comment]
The second option works! If you push up a 3.5.0 branch you can change this PR to be against it.

**Review by tmooney — APPROVED** [review]
+1 ⛳️

**Review by jasonwalker80 — APPROVED** [review]
+1

**Pull request merged**